docs/DeveloperGuide.adoc

```diff
@@ -242,10 +242,11 @@ To avoid writing a long command line when ordering a long list of dishes, a draf
 
 * When the ordering draft is initialized or cleared, the member `draftOrder` in `Rms.txt` is loaded with a new object
 of class `Order`. It is initialize with an empty list of dish and a pseudo-empty customer field that is not `null`.
-* By using the draft order command classes, the ordering draft can be edited.
-* Once an order is confirmed, the time is recorded and the total price is calculated to create a new `Order` object
-that has the same customer and list of dishes fields as the ordering draft to add to the order list, and the ordering
-draft is cleared afterward.
+* By using the draft order command classes, the ordering draft can be edited (e.g. picking the customer from the member
```

Comment:
This is a usage example but I think it would be better if you could detail the usage. I would suggest it would be good to follow the way the 'undo/redo' feature in AB4 Dev Guide (under Implementation) has explained their usage scenario. It is ok if you do not add the additional diagrams in every step. But if you could explain the user-system interaction of your feature in the way they did, it would be great

Reply:
The feature is not complicated enough to have a dedicated section for explaining how it works.